bamazonCustomer.js

```diff
@@ -0,0 +1,166 @@
+var mysql = require("mysql");
+var inquirer = require("inquirer");
+
+
+var connection = mysql.createConnection({
+  host: "localhost",
+  port: 3306,
+
+  // Your username
+  user: "root",
+
+  // Your password
+  password: "",
+  database: "bamazonDB"
+  // database: "ice_creamDB"
+});
+
+connection.connect(function(err) {
+  if (err) {
+    throw err;
+  };
+  console.log("connected as id " + connection.threadId);
+  afterConnection();
+});
+
+function afterConnection() {
+
+  connection.query("SELECT * FROM products", function(err, res) {
+    if (err) {
+      throw err;
+    };
+
+    startBamazon(res);
+
+  });
+
+}
+
+
+function startBamazon(res) {
+  displayAllItem(res);
+  inquirer.prompt([
+    {
+      type: 'input', 
+      name: "id",
+      message: "Enter product ID for item that you want to purchase?",
+      validate: function(value) {
+        if(value <= 10) {
```

Comment:
I would add extra validation here, like making sure that what input is givin is a number